psa-crypto-sys/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "psa-crypto-sys"
-version = "0.9.0"
+version = "0.9.1"
```

Comment:
Do we need to bump this? Is it for the `build.rs` change?

Reply:
This new functionality is required for can-do-crypto for TS and mbed-crypto providers. So, I was planning to bump version here and in parsec and parsec-interface cargo.toml. If I can avoid the bump it would make my life easier.

Reply:
I don't think you need to bump the `-sys` crate unless you want to release something specific. Might be good to release that change from late last week, though!

Reply:
removed the bump from `-sys`